crates/autopilot/src/solvable_orders.rs

```diff
@@ -604,19 +611,26 @@ async fn update_task(
     }
 }
 
-fn get_orders_with_native_prices(
+async fn get_orders_with_native_prices(
     orders: Vec<Order>,
     native_price_estimator: &CachingNativePriceEstimator,
     metrics: &Metrics,
+    additional_tokens: &[H160],
```

Comment:
Instead of passing the tokens it seems more natural to pass the cow amms directly. That way you also don't have to collect the cow amm traded tokens into a vector before hand.

Reply:
Good point. 👍